src/xToken.ts

```diff
@@ -732,4 +738,27 @@ export class XToken {
         return mintXU3LP(symbol, tradeWithEth ? 1 : 0, value, this.provider)
     }
   }
+
+  public async stakeXtk(amount: string) {
+    return stakeXtk(this.provider, amount)
+  }
+
+  public async unstakeXXtkA(amount: string) {
+    return unstakeXXtkA(this.provider, amount)
+  }
+
+  public async getXtkStakeHistory(
+    account: string,
```

Comment:
Remove the `account` as input argument. We should be fetching the user address off of `provider` as well as add the error checks if it isn't a valid address. Refer to an example usage here: https://github.com/xtokenmarket/js/blob/main/src/xToken.ts#L607-L611